libs/styles/styles.css

```diff
@@ -1,134 +1,586 @@
-/*** Variables ***/
+/* stylelint-disable comment-empty-line-before */
+/* stylelint-disable custom-property-empty-line-before */
 
- :root {
+:root {
   --global-height-nav: 64px;
   --global-height-breadcrumbs: 33px;
   /* stylelint-disable-next-line custom-property-pattern */
   --global-height-navPromo: 65px;
   --feds-totalheight-nav: calc(var(--feds-height-nav, --global-height-nav) + var(--feds-height-breadcrumbs, --global-height-breadcrumbs));
 
-  /* Colors */
-  --link-color: #035fe6;
-  --link-hover-color: #136ff6;
+  /* Fonts */
+  --body-font-family: 'Adobe Clean', adobe-clean, 'Trebuchet MS', sans-serif;
+  --headline-font-family: 'Adobe Headline', adobe-headline, 'Adobe Clean', sans-serif; /* new */
+  --fixed-font-family: 'Roboto Mono', menlo, consolas, 'Liberation Mono', monospace;
+
+  /* Milo Color Specs */
+  --color-white: var(--s2-white); 
+  --color-black: var(--s2-black); 
+  --color-gray-100: var(--s2-gray-100,#F8F8F8);
```

Comment:
I don't remember where we landed on the default values. Does it make sense to continue defining them? If we start using S2, we'd get all of the variables, so the default would never be used, right?

If, for some reason, we need them, nit: space after the comma, same for L21.

Reply:
I think we can get rid of the default values at this point. These were only here to keep note of for reference purposes.